docs/topics_5/web-add-project.adoc

```diff
@@ -1,16 +1,17 @@
 // Module included in the following assemblies:
 // * docs/web-console-guide_5/master.adoc
 [id='add_project_{context}']
-= Adding a project
+= Creating a project
 
-When you first access the {WebName}, you will be prompted to set up a new project.
+You begin the process of defining a project by creating it.
 
-.Home
-image::web-no-projects.png[Home: no projects]
+.Procedure
 
-Click the *New Project* button to launch the wizard to set up a new migration project.
+. On the *Projects* page, click *Create project*.
++
+.Creating a project
+image::web-add-project_51.png[Creating a project]
 
-.Adding a project
-image::web-add-project.png[Adding a project]
-
-Specify a name and an optional description for this project and click *Next* to continue.
+. Enter a unique name for the project.
+. Optional: Enter a description.
+. Click *Next*.
```

Comment:
That's it? That's the end of the procedure? What does the user see after click "Next"? Are you sure this procedure is finished? Usually "Next" takes you to another page in a wizard.

What about a confirmation step? You learned that in the Minimalism workshops, right?